erpnext/translations/de.csv

```diff
@@ -6003,7 +6003,7 @@ CN,CN,
 DE,DE,
 ES,ES,
 FR,FR,
-IN,IM,
+IN,Ein,
```

Comment:
hey can you give some instances where **Ein** is used as IN (context IN/OUT) ? From everything I can find online **IN** translates to IN (english) and **IM** translated to IN THE (english)

Reply:
@marination  on new-employee-checkin [example : stamp in = einstempeln   stamp out = ausstempeln] in = ein, out = aus
![grafik](https://user-images.githubusercontent.com/48571007/179797131-afb2fb56-4970-497d-9606-a1e551388d6b.png)

Reply:
Had to check if IN was used elsewhere in other context. Nothing so far so this is good to go